.github/workflows/ci.yml

```diff
@@ -27,7 +27,8 @@ jobs:
         if: runner.os == 'Linux'
         uses: easimon/maximize-build-space@master
         with:
-          root-reserve-mb: 2048
+          # Leaving 20 GB for rust crates download and installation under `/tmp`
+          root-reserve-mb: 20480
```

Comment:
I encountered running out of space error because rust crates are downloaded under `/tmp` dir for installation. So here, we increase the space reserved for root dir.